docs/core-concepts/custom-resources.md

```diff
@@ -1,14 +1,17 @@
 ---
 title: Custom Trace Resource
-weight: 200
+sidebar_position: 200
 description: >
   Basic usage of the Trace resource
 ---
 
-> [!WARNING]
-> We are currently removing the Trace resource and using a [gRPC
-> API](../reference/grpc.md) instead. Once done, the Trace resource will be
-> deprecated.
+:::warning
+
+We are currently removing the Trace resource and using a [gRPC
+API](../reference/grpc.md) instead. Once done, the Trace resource will be
+deprecated.
+
+:::
```

Comment:
Isn't the previous formatted in a good way? From https://polite-mushroom-00d947803-53.westeurope.1.azurestaticapps.net/docs/latest/core-concepts/custom-resources it seems the format is fine, so if possible I'd like to stick with it so it's also formatted on Github.

Reply:
This new format is required for docusaurus. The reason it works in that deployment is because the admonitions are converted (using this [script](https://github.com/inspektor-gadget/website/blob/4533143dfbcd979c6e30905e535f613ca78a1923/tools/docs-fetcher.py#L79)). But when linking the latest docs from the source for the preview we want it to be fully in docusaurus format